src/FirePokemon.java

```diff
@@ -0,0 +1,45 @@
+public class FirePokemon extends Pokemon {
+    private int level;
+    private int flamesLeft;
+
+    public FirePokemon(String name, int hp, int damage, String action) {
+        super(name, hp, "Red");
+        super.setDamage(damage);
+        super.setAction(action);
+        this.level = 1;
+        this.flamesLeft = 3;
+    }
+
+    @Override
+    public void makeSound() {
+        System.out.println("BURN BABY B U R N ! ! !");
+    }
+
+    public void flameThrower() {
+        if (this.flamesLeft > 0) {
```

Comment:
mooi gebruik gemaakt van de al opgedane kennis door de if statement toe te passen. Extra tipje verminder het aantal flames in je methode voor deze sluit met flamesLeft= flamesLeft - 1